test/config/filters/WhitelistFilterSpec.scala

```diff
@@ -20,15 +20,20 @@ import config.AppConfig
 import play.api.{Application, Configuration, Environment}
 import org.scalatestplus.play.PlaySpec
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.Mode.Test
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.mvc.Results.Ok
 import play.api.mvc.{Action, Call}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import uk.gov.hmrc.play.bootstrap.config.{RunMode, ServicesConfig}
 
 class WhitelistFilterSpec extends PlaySpec with GuiceOneServerPerSuite {
 
-  lazy val mockConfig = new MockAppConfig(app.configuration)
+  //val servicesConfig = app.injector.instanceOf[ServicesConfig]
```

Comment:
If there is no extra reason to keep this line commented, then I would remove it.

Reply:
Yeah, good spot. I had a problem with this tests. I tried few different approach, just forgot about cleaning code after everything.